libs/books/data-access/src/lib/+state/reading-list.effects.ts

```diff
@@ -54,9 +56,51 @@ export class ReadingListEffects implements OnInitEffects {
     )
   );
 
+  showSnackBarOnAdd$=createEffect(() => 
+    this.actions$.pipe(
+      ofType(ReadingListActions.confirmedAddToReadingList),
+      concatMap((action) => {
+        const typeItem:ReadingListItem = {
+          ...action.book,
+          bookId: action.book.id
+        }
+        if(action.add === false){
+          return this.snackBar.open(
+            `Added ${action.book.title} to reading list`,'Undo', { duration: 2000}
```

Comment:
2000 is very less value for snackbar display at least it should have minimum value 3000

Reply:
Changed min value to 3000